.github/CODEOWNERS

```diff
@@ -4,8 +4,9 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-# The @googleapis/api-bigquery is the default owner for changes in this repo
-*               @googleapis/api-bigquery @googleapis/yoshi-python
 
-# The python-samples-reviewers team is the default owner for samples changes
-/samples/   @googleapis/python-samples-owners
+# @googleapis/yoshi-python and @googleapis/api-bigquery are the default owners for changes in this repo
+*     @googleapis/yoshi-python @googleapis/api-bigquery
+
+# @googleapis/python-samples-owners and @googleapis/api-bigquery are the default owners for samples changes
```

Comment:
Could the synthtool template for CODEOWNERS note that the file is generated and that edits should be made to `.repo-metadata.json`?

Reply:
Will be added in https://github.com/googleapis/synthtool/pull/1201